Extensions/Today/ButtonWithSublabel.swift

```diff
@@ -26,26 +26,32 @@ class ButtonWithSublabel: UIButton {
         self.titleLabel?.removeFromSuperview()
         addSubview(self.label)
         addSubview(self.subtitleLabel)
-
+        buttonImage.adjustsImageSizeForAccessibilityContentSizeCategory = true
+        buttonImage.contentMode = .scaleAspectFit
+        
         buttonImage.snp.makeConstraints { make in
-            make.centerY.left.equalTo(10)
-            make.width.equalTo(TodayUX.copyLinkImageWidth)
+            make.left.equalTo(10)
+//            make.top.equalTo(self.label.snp.top)
+            make.centerY.equalTo(self.label.snp.firstBaseline)
```

Comment:
@noorhashem Little confused as why the centerY is towards label firstBaseline.
Are you trying to center it with the label or literally center on top?

Reply:
@nbhasin2  when you resize the label the icon seems off when it's centerY is spaced with 10 as it used to be, so I made it relative to the baseline of the label